src/test/operators/table_scan_between_test.cpp

```diff
@@ -26,7 +26,7 @@ class TableScanBetweenTest : public TypedOperatorBaseTest {
     // ...
     // 30.25         10
     //
-    // As the first column is type casted, it contains 10 for an int column, the string "10.25" for a string column etc.
+    // As the first column is TYPE CASTED, it contains 10 for an int column, the string "10.25" for a string column etc.
```

Comment:
Why did you capitalize this?

Reply:
It took me some time to realize its meaning, because it turned out to be very important for the test results. Otherwise I would add another comment starting with 'Attention', but I found it already described enough. One just needs to read it.
Whatever you like more, I am flexible, but I would like to highlight this fact